src/core.js

```diff
@@ -60,7 +60,7 @@ let readLater = (function(storageObject) {
       return false;
     }
     for (let key in syncItem) {
-      if (!('title' in syncItem[key]) || !('timestamp' in syncItem[key])) {
+      if (typeof syncItem[key].title === "undefined" || typeof syncItem[key] === "undefined") {
```

Comment:
The order of checking should be reversed because if the object is not defined then there is no point checking for title. Also do update the code to 4.0.3 which is the latest API with similar fix.

Reply:
You seem to be missing the point.
`'title' in syncItem[key]` is invalid JS syntax.

Reply:
But, yes, I agree on the order. I'll cancel this PR.